_docs-sources/guides/stay-up-to-date/cis/cis-1.5.0/deployment-walkthrough/step-4-verify-the-code-changes.md

```diff
@@ -0,0 +1,100 @@
+---
+sidebar_label: Verify the code changes
+---
+
+# Step 4: Verify the code changes
+
+:::caution
+
+We strongly recommeded that you verify the changes that have been made before executing Terraform/Terragrunt `apply`
+
+:::
+
+Upgrading to CIS AWS Foundations Benchmark v1.5.0 includes migrating to our new CIS RDS module, this breaking change will result in data loss if not updated correctly.
+
+We strongly recommend that you run `terraform diff` and verify that any RDS resources you have will be "updated in-place" and not destroy and new resources created.
+
+## The migrations were correctly executed
+
+If the migrations have been correctly executed then running `terraform diff` will output a message similar to this one:
```

Comment:
There's no `terraform diff` command. Do you mean to run `terraform plan`? Or `terragrunt plan`? And if so, in which folder(s)?

Reply:
@brikis98 I'll change it to "`terraform plan` or `terragrunt plan`"